test_unishox2.c

```diff
@@ -403,228 +445,228 @@ if (argv >= 2 && strcmp(args[1], "-t") == 0) {
     if (!test_ushx_cd("HELLO WORLD HELLO WORLD", preset)) return 1;
 
     // Numbers
-    if (!test_ushx_cd("Hello1", preset)) return 1;
-    if (!test_ushx_cd("Hello1 World2", preset)) return 1;
-    if (!test_ushx_cd("Hello123", preset)) return 1;
-    if (!test_ushx_cd("12345678", preset)) return 1;
-    if (!test_ushx_cd("12345678 12345678", preset)) return 1;
-    if (!test_ushx_cd("HELLO WORLD 1234 hello world12", preset)) return 1;
-    if (!test_ushx_cd("HELLO 234 WORLD", preset)) return 1;
-    if (!test_ushx_cd("9 HELLO, WORLD", preset)) return 1;
-    if (!test_ushx_cd("H1e2l3l4o5 w6O7R8L9D", preset)) return 1;
-    if (!test_ushx_cd("8+80=88", preset)) return 1;
+    if (preset != 1 && !test_ushx_cd("Hello1", preset)) return 1;
+    if (preset != 1 && !test_ushx_cd("Hello1 World2", preset)) return 1;
```

Comment:
I appreciate the idea of testing the presets as well and completing the implementation of presets.